.circleci/config.yml

```diff
@@ -20,6 +20,8 @@ jobs:
           mapping: |
             kolena/classification/.* classification true
             tests/integration/classification/.* classification true
+            kolena/_experimental/.* _experimental true
```

Comment:
Should we just use `workflow` as this is built on top of generic workflow codes, as a result, when generic workflow codes are updated, `_experimental` tests should also be run?